frontend/src/serviceprovider/profile/Profile.scss

```diff
@@ -0,0 +1,159 @@
+/* Kontener bazowy */
+.baseprofile{
+  // Marginesy
+  margin-top: 50px;
+  margin-right: 20%;
+  margin-left: 20%;
+  margin-bottom: 50px;
+  // Pozycjonowanie
+  display: flex;
+  flex-direction: column;
+  // Ramka
+  border: 2px ridge black;
+}
+
+/* Kontener na nazwę i zdjęcie */
+.salon-name-photo{
+  display: flex;
+  flex-direction: column;
+}
+
+/* Kontener na nazwę salonu */
+.salon-name{
+  // Tekst
+  font-size: 5vw;
+  font-weight: bold;
+  // Położenie
+  display: flex;
+  flex-direction: column;
+}
+
+/* Kontener na zdjęcie salonu */
+.salon-photo{
+  // Proporcja zdjęcia
+  aspect-ratio: 16 / 9;
+}
+
+/* Przycisk przy zdjęciu */
+.salon-photo-button{
+  // Kolory
+  background: #F85F6A;
+  color: white;
+  // Ramka
+  border: 1px solid #F85F6A;
+  border-radius: 4px;
+  // Pozycjonowanie
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  // Czcionka
+  font-size: 2vw;
+  // Cień
+  //box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
```

Comment:
Nie trzymajmy zakomentowanego kodu w repozytorium, skoro nie jest potrzebny to można usunąć.